alicloud/resource_alicloud_oss_bucket.go

```diff
@@ -204,8 +204,10 @@ func resourceAlicloudOssBucket() *schema.Resource {
 				Computed: true,
 			},
 			"storage_class": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:         schema.TypeString,
+				Default:      oss.StorageStandard,
+				Optional:     true,
```

Comment:
Can the storage_class be modified? If not, please set `ForceNew` to the parameter.

Reply:
add the ForceNew to the storage_class parameter